cloud/availability.rst

```diff
@@ -163,6 +166,66 @@ Such crashes are a nightmare. But crash recovery is a way to make sure
 you sleep well at night by making sure none of your or your customers
 data is lost and your downtime is minimal.
 
+.. _cloud_forking:
+
+Forking
+=======
+
+Forking a Citus Cloud formation makes a copy of the cluster's data at an instant in time and produces a new formation in precisely that state. It allows you to change, query, or generally experiment with production data in a separate protected environment. Fork creation runs quickly, and you can do it as often as you want without causing any extra load on the original cluster. This is because forking doesn't query the cluster, rather it taps into the write-ahead logs for each database in the formation.
```

Comment:
Maybe say "at the current point in time" instead of "instant in time" (which a non-native speaker might read as "instantly")